skyrim-platform/src/tilted/ui/DX11RenderHandler.cpp

```diff
@@ -72,17 +74,31 @@ void DX11RenderHandler::Render(
       static_assert(
         std::is_same_v<std::decay_t<decltype(textToDraw.string.c_str()[0])>,
                        wchar_t>);
-      auto origin = DirectX::SimpleMath::Vector2(m_pSpriteFont->MeasureString(
-                      textToDraw.string.c_str())) /
-        2;
-
-      DirectX::XMVECTORF32 color = { static_cast<float>(textToDraw.color[0]),
-                                     static_cast<float>(textToDraw.color[1]),
-                                     static_cast<float>(textToDraw.color[2]),
-                                     static_cast<float>(textToDraw.color[3]) };
-      m_pSpriteFont->DrawString(
-        m_pSpriteBatch.get(), textToDraw.string.c_str(),
-        DirectX::XMFLOAT2(textToDraw.x, textToDraw.y), color, 0.f, origin);
+
+      std::wstring widestrFontName =
+        std::wstring(textToDraw.fontName.begin(), textToDraw.fontName.end());
+      
+      const wchar_t* fontPath = widestrFontName.c_str();
+
+      DirectX::SpriteFont m_pSpriteFont =
+        DirectX::SpriteFont(m_pDevice.Get(), fontPath);
+        
+      auto origin = DirectX::SimpleMath::Vector2(m_pSpriteFont.MeasureString(textToDraw.string.c_str())) / 2;
```

Comment:
Looks like clang-format will reformat this

Reply:
fixed